testenv-deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+usage="usage: $(basename "$0") [-h] LABEL [API_URI]"
+
+if [ "$1" == "-h" ]; then
+    echo $usage 
+    exit 0
+fi
+
+if [ $# -lt 1 ]; then
+    echo "Error: Missing argument: LABEL"
+    echo $usage 
+    exit 1
+fi
+
+DOMAIN="$1.storefront.saleor.rocks"
+API_URI=${2:-$API_URI}
+
+if [ -z $API_URI ]; then
+    echo "Error: Missing argument or env variable: API_URI"
+    echo $usage 
+    exit 1
+fi
+
+# API_URI=$API_URI npm run build \
```

Comment:
```suggestion
```

Reply:
I've removed this file. It was added accidentally